Gemfile

```diff
@@ -36,35 +36,37 @@ gem 'jquery-timepicker-rails'
 gem 'bootstrap-datepicker-rails'
 gem 'twitter-bootstrap-rails-confirm'
 
+gem 'responders', '~> 2.0'
+
 #RMagick is used for image resizing and processing
-gem 'rmagick', :require => 'rmagick', :platforms => :ruby
+gem 'rmagick', require: 'rmagick', platforms: :ruby
 
 # Attachable does all the file work.
 gem 'attachable'
 
-gem 'devise', '~> 3'
+gem 'devise', '~> 3.5'
```

Comment:
do we want minor versions to float, or just tiny? ~> 3.5 means <4, whereas ~> 3.5.3 would mean <3.6